astropy/modeling/physical_models.py

```diff
@@ -10,11 +10,12 @@
 
 from astropy import constants as const
 from astropy import units as u
+from astropy import cosmology
 from astropy.utils.exceptions import AstropyUserWarning
-from .core import Fittable1DModel
+from .core import Fittable1DModel, Fittable2DModel
 from .parameters import Parameter, InputParameterError
 
-__all__ = ["BlackBody", "Drude1D", "Plummer1D"]
+__all__ = ["BlackBody", "Drude1D", "Plummer1D", "NFW2D", "NFW1D"]
```

Comment:
Maybe switch the order to `"NFW1D", "NFW2D"` since that's the order in the module?